docker/1.4.1/py2/Dockerfile.gpu

```diff
@@ -0,0 +1,64 @@
+FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu16.04
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
+    fakeroot \
+    ca-certificates \
+    dpkg-dev \
+    g++ \
+    python-dev \
+    openjdk-8-jdk-headless \
+    curl \
+    vim \
+    && rm -rf /var/lib/apt/lists/* \
+    && cd /tmp \
+    && curl -O https://bootstrap.pypa.io/get-pip.py \
+    && python get-pip.py
+
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python2.7 1
+
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends \
+    libopencv-dev \
+    build-essential \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /
+
+COPY dist/sagemaker_mxnet_serving_container.tar.gz /sagemaker_mxnet_serving_container.tar.gz
+
+RUN pip install --no-cache mxnet-cu100mkl==1.4.1 \
+                           mxnet-model-server==1.0.5 \
```

Comment:
Just curious, why not use "args" as you used for other Dockerfiles?

Reply:
the py2 and EI Dockerfiles are done in the style of our old Dockerfiles. Given the hardcoded version in the directory path, there's no intent of overriding these versions when we're building from those Dockerfiles. The py3 ones with args were done in the style of the DLC Dockerfiles, which is why they're different.